pennylane/templates/transforms.py

```diff
@@ -0,0 +1,20 @@
+from functools import wraps
+from pennylane.tape.tapes import QuantumTape
+
+
+def adjoint(fn):
+    """ "Create a new method that applies the adjoint tape of `fn`.
+

```

Comment:
Don't forget to include `Args:` and `Returns`:

Reply:
(bump)

Reply:
Added (I did add it before but for to git add it)